packages/compass-e2e-tests/helpers/commands/connect-with-connection-string.ts

```diff
@@ -36,24 +36,26 @@ export async function connectWithConnectionString(
   );
   await browser.doConnect(connectionStatus, timeout);
 
-  // Ideally this would be part of browser.doConnect() and
-  // browser.waitForConnectionResult(), but that requires the context of
-  // connectionName there. Which is easy enough for connectWithConnectionString,
-  // but not so easy right now for connectWithConnectionForm.
-  if (TEST_MULTIPLE_CONNECTIONS) {
-    // There's a potential problem here: The list of connections is virtual, so
-    // the new connection might not be rendered. By searching for it if possible
-    // we can guarantee that it will be on screen. The search box won't exist
-    // for the first connection, but that's OK because once connected it will be
-    // the only connection so should be rendered.
-    if (await browser.$(Selectors.SidebarFilterInput).isExisting()) {
-      await browser.clickVisible(Selectors.SidebarFilterInput);
-      await browser.setValueVisible(Selectors.SidebarFilterInput, '');
-    }
+  if (connectionStatus === 'success') {
```

Comment:
could the part that is only for successful connections be separated into it's own helper? `connectWithConnectionString` is becoming somewhat fuzzy